### PR TITLE
New Icon for Vector.AngleAboutAxis

### PR DESCRIPTION
**Pull request by YanmengLi**
### Purpose
Reference: [MAGN-9828](http://adsk-oss.myjetbrains.com/youtrack/issue/MAGN-9828)
This PR is to add the missing icon for Vector.AngleAboutAxis node.
vector angleaboutaxis

### Declarations

Check these if you believe they are true

- [ ] The code base is in a better state after this PR
- [ ] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [ ] The level of testing this PR includes is appropriate
- [ ] User facing strings, if any, are extracted into `*.resx` files
- [x] All tests pass using the self-service CI.
- [x] Snapshot of UI changes, if any.

### Reviewers

@Racel 

### FYIs

@riteshchandawar

### CODE REVIEW

**Comment by Racel** [comment]
@YanmengLi please add more description to the purpose and check off the items in the declaration section. We would like  to make sure that the PR templates are filled out correctly so that we have an accurate record. Also, please make sure to add an image as this is a UI change.

**Comment by aosyatnik** [comment]
Hi, @YanmengLi , and, please, add/update icon in src\Resources\ProtoGeometry\SmallIcons

**Comment by YanmengLi** [comment]
Updated. Thanks @Racel for checking.

**Comment by aosyatnik** [comment]
@YanmengLi  and in Large folder too :) src\Resources\ProtoGeometry\LargeIcons

**Comment by YanmengLi** [comment]
Added

File: src/Libraries/GeometryUIWpf/GeometryUIWpf.csproj [hunk 1]
```diff
@@ -126,6 +126,9 @@
     </EmbeddedResource>
     <EmbeddedResource Include="ProtoGeometryImages.resx" />
   </ItemGroup>
+  <ItemGroup>
+    <Folder Include="Resources\" />
```
Comment (aosyatnik) [inline src/Libraries/GeometryUIWpf/GeometryUIWpf.csproj]: Why do we need to include this folder?

**Comment by YanmengLi** [comment]
Removed

**Comment by YanmengLi** [comment]
Please do not merge this first, I will rework the images to make them smaller. Thanks.

**Comment by YanmengLi** [comment]
@Racel The images are updated. Thanks for helping check. Will save the ai files to Google drive tomorrow.

**Pull request merged**